packages/extension-koni/public/locales/en/translation.json

```diff
@@ -1,5 +1,163 @@
+  "settings.Screen.settingsEntry.Item.manageAddressBook": "Manage address book",
+  "settings.Screen.settingsEntry.Item.manageTokens": "Manage tokens",
+  "settings.Screen.settingsEntry.SectionTitle.communitySupport": "Community & support",
+  "settings.Screen.settingsEntry.Item.contactSupport": "Contact support",
+  "settings.Screen.settingsEntry.Item.userGuide": "User guide",
+  "settings.Screen.settingsEntry.Item.requestFeature": "Request a feature",
+  "settings.Screen.settingsEntry.Item.aboutSubwallet": "About SubWallet",
+  "settings.Screen.settingsEntry.Button.lock": "Lock",
+
+  "settings.Screen.generalSetting.title": "General settings",
+  "settings.Screen.missionPools.title": "Mission Pools",
+  "settings.Screen.manageWebsiteAccess.title": "Manage website access",
+  "settings.Screen.manageNetworks.title": "Manage networks",
+  "settings.Screen.generalSetting.walletTheme": "Wallet theme",
+  "settings.Screen.generalSetting.Item.currency": "Currency",
+  "settings.Screen.generalSetting.Item.language": "Language",
+  "settings.Screen.generalSetting.Item.notifications": "Notifications",
+  "settings.Screen.securitySetting.title": "Security settings",
+  "settings.Screen.securitySetting.Item.changePassword": "Change wallet password",
+  "settings.Screen.securitySetting.Item.autoLock": "Extension auto lock",
+  "settings.Screen.securitySetting.Item.authPassword": "Authenticate with password",
+  "settings.Screen.securitySetting.Item.phishingDetection": "Advanced phishing detection",
+  "settings.Screen.securitySetting.Item.cameraAccess": "Camera access for QR",
+  "settings.Screen.websiteAccess.searchPlaceHolder": "Search or enter a website",
+  "settings.Screen.manageNetworks.searchPlaceHolder": "Search networks",
+  "settings.Screen.manageTokens.searchPlaceHolder": "Search token",
+
+  "settings.Screen.generalSetting.currency.selectCurrency.title": "Select a currency",
+  "settings.Screen.generalSetting.language.selectLanguage.title": "Language",
+  "settings.Screen.generalSetting.notification.selectPlatform.title": "Notifications",
+  "settings.Screen.generalSetting.Selector.currency.title": "Select a currency",
+  "settings.Screen.generalSetting.Selector.currency.searchPlaceHolder": "Search currency",
+  "settings.security.Screen.changePassword.title": "Change password",
+  "settings.security.Screen.changePassword.bodyTitle": "Change your password",
+  "settings.security.Screen.changePassword.Input.currentPassword.placeHolder": "Current password",
+  "settings.security.Screen.changePassword.FormRules.currentPassword": "Password is required",
+  "settings.security.Screen.changePassword.Input.newPassword.placeHolder": "New password",
+  "settings.security.Screen.changePassword.Input.confirmPassword": "Confirm new password",
+  "settings.security.Screen.changePassword.confirmAgreement": "I understand that SubWallet can’t recover the password.",
+  "settings.security.Screen.changePassword.link.learnMore": "Learn more",
```

Comment:
@dungnguyen-art change to settings.security.Screen.changePassword.Link.learnMore